Example/navigation/index.js

```diff
@@ -35,57 +30,56 @@ const Background = ({ index }) => (
   />
 );
 
-class DetailsScreen extends React.Component {
+const DetailsScreen = ({ navigation, route }) => {
   animvalue = new Animated.Value(0);
```

Comment:
Same for the line below

```suggestion
  const animvalue = new Animated.Value(0);
```